Version Migration Guides/V3_to_V4_ParameterMigrationGuide.md

```diff
@@ -0,0 +1,178 @@
+| mod_loc | Removed as with new object oriented structure type can support this when new sensor types are properly implemented |
+| cost: per_hour | Removed as it was misleading, LDAR-SIm functions on a day timescale |
+| scheduling: LDAR_crew_init_location | Removed all routing parameters as it was not fully functional |
+| scheduling: home_base-files | Removed all routing parameters as it was not fully functional |
+| scheduling: travel_speeds | Removed all routing parameters as it was not fully functional |
+| scheduling: min_time_bt_surveys | New scheduling algorithm uses queue based scheduling, removing the need for this parameter |
+| follow_up: instant_threshold_type | [Instant threshold](../USER_MANUAL.md#instant_threshold) is now fixed to an absolute threshold|
+| follow_up: min_followups| To be reimplemented if needed|
+| follow_up: min_followup_type| To be reimplemented if needed|
+| follow_up: min_followup_days_to_end| To be reimplemented if needed|
+| follow_up: instant_threshold_type | [Threshold](../USER_MANUAL.md#threshold-mobile-parameter) is now fixed to an absolute threshold|
+
+#### Methods: New Parameters
+
+| New Parameter | Notes                                                                   |
+|--------------------|-----------|
+| sensor: quantification_error: quantification_parameters | See [relevant](../USER_MANUAL.md#quantification_parameters) section in the user manual|
+| sensor: quantification_type: quantification_parameters | See [relevant](../USER_MANUAL.md#quantification_type) section in the user manual|
+| rolling_average : small_window | stationary methods only. See [relevant](../USER_MANUAL.md#small_window-stationary-parameter) section in the user manual|
+| rolling_average : large_window | stationary methods only. See [relevant](../USER_MANUAL.md#large_window-stationary-parameter) section in the user manual|
+| rolling_average : small_window_threshold | stationary methods only. See [relevant](../USER_MANUAL.md#small_window_threshold-stationary-parameter) section in the user manual|
+| rolling_average : large_window_threshold | stationary methods only. See [relevant](../USER_MANUAL.md#large_window_threshold-stationary-parameter) section in the user manual|
+
+## Input Files
+
+### Unchanged Input Files
+
+The weather file has remained unchanged from V3 to V4.
+
+### Infrastructure
+
+A new, much more granular infrastructure design has been introduced. See [infrastructure](../USER_MANUAL.md#infrastructure) and [virtual world defining files](../USER_MANUAL.md#10-virtual-world-defining-files) sections in the user manual for more information.
+
+### Emissions File (New to V4)
+
+New to V4 is a new file called the Emissions Fie. This file is mandatory to run any V4 simulations. Read more about it in the User Manual [here](../USER_MANUAL.md#emissions-file). It must be set in the virtual world parameters, see the relevant section [here](../USER_MANUAL.md#emissions_file).
```

Comment:
Linter will likely complain about no new lines at the end of the file.

Reply:
hmm there is an empty line, might be a git diff view thing?